src/annotator/guest.js

```diff
@@ -210,23 +208,10 @@ export default class Guest extends Delegator {
   }
 
   addPlugin(name, options) {
-    if (this.plugins[name]) {
-      console.error('You cannot have more than one instance of any plugin.');
-    } else {
-      const Klass = this.options.pluginClasses[name];
-      if (typeof Klass === 'function') {
-        this.plugins[name] = new Klass(this.element[0], options);
-        this.plugins[name].annotator = this;
-        this.plugins[name].pluginInit?.();
-      } else {
-        console.error(
-          'Could not load ' +
-            name +
-            ' plugin. Have you included the appropriate <script> tag?'
```

Comment:
This is a vestige of the Annotator project where consumers could specify their own plugins. In Hypothesis this doesn't happen. Any errors here indicate a programming error elsewhere in annotator and I'd prefer that they just fail loudly.